west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 44ede1e459c061fb1f7b65431535d6b66654ec78
+      revision: 85445474f29eed2ad09062456aaeb1dfc204e8cf
```

Comment:
This should be `86893246053c50d34618f09ec6722cae8ba19472` (https://github.com/nrfconnect/sdk-zephyr/commit/86893246053c50d34618f09ec6722cae8ba19472)